package.json

```diff
@@ -66,12 +68,15 @@
     "loader.js": "^4.2.3"
   },
   "engines": {
-    "node": "6.* || >= 7.*"
+    "node": "8.* || >= 10.*"
   },
   "ember-addon": {
     "configPath": "tests/dummy/config"
   },
   "publishConfig": {
     "@addepar:registry": "https://registry.npmjs.org/"
+  },
+  "resolutions": {
+    "ember-cli-page-object": "1.15.1"
```

Comment:
Instead of this can https://github.com/pzuraq/ember-classy-page-object simply be bumped to v0.6.1? I believe as of v0.6.0 the version of ember-cli-page-object is bumped.

Reply:
Unfortunately no, at least not simply. I've spent some time trying to figure out the problem and couldn't solve it in a timely manner (#69 was one attempt). While discussing w/ @bantic, he mentioned that he also got blocked when looking at the upgrade (See #65).

Reply:
@cyril-sf what is the blocker?